dodo.py

```diff
@@ -28,6 +47,7 @@ def task_up():
                 "default": 8888,
                 "help": "Specify the AiiDAlab host port.",
             },
+            _REGISTRY_PARAM,
```

Comment:
I am probably nitpicking here, but if you decided to place registry param as a global variable, it probably makes sense to do the same for the port.

Reply:
The port is only used for the up task whereas the _REGISTRY_PRAM is used in all commands.

Reply:
ok, sounds good.